### ISSUE TITLE
Generated Excel spreadsheet is corrupt on AWS m7i.16xlarge with Ubuntu 22.04 LTS

### ISSUE THREAD

**Issue by sanchitintel** [intel/svr-info#20]
The generated Excel spreadsheet is corrupt on AWS m7i.16xlarge instances with Ubuntu 22.04.

I tried `sudo` as well, but same result.
The generated webpage renders fine, though.

<img width="619" alt="image" src="https://github.com/intel/svr-info/assets/90872293/91361ea4-5e9e-4296-8ce6-08f3aad87402">

<img width="370" alt="image" src="https://github.com/intel/svr-info/assets/90872293/54abae55-1259-45b1-a52e-b08ff586bfdd">

Excel Logs
```
<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<recoveryLog xmlns="http://schemas.openxmlformats.org/spreadsheetml/2006/main"><logFileName>error120440_01.xml</logFileName><summary>Errors were detected in file 'filename.xlsx'</summary><repairedParts><repairedPart>Repaired Part: /xl/worksheets/sheet1.xml part with XML error.  HRESULT 0x8000ffff Line 2, column 0.</repairedPart><repairedPart>Repaired Part: /xl/worksheets/sheet2.xml part with XML error.  HRESULT 0x8000ffff Line 2, column 0.</repairedPart><repairedPart>Repaired Part: /xl/worksheets/sheet3.xml part with XML error.  HRESULT 0x8000ffff Line 2, column 0.</repairedPart><repairedPart>Repaired Part: /xl/worksheets/sheet4.xml part with XML error.  Catastrophic failure Line 2, column 955.</repairedPart><repairedPart>Repaired Part: /xl/worksheets/sheet5.xml part with XML error.  Catastrophic failure Line 2, column 457.</repairedPart><repairedPart>Repaired Part: /xl/worksheets/sheet6.xml part with XML error.  Catastrophic failure Line 2, column 577.</repairedPart></repairedParts></recoveryLog>
```

**Comment by harp-intel** [comment]
Bug confirmed in 2.7.1. Root cause is: https://github.com/qax-os/excelize/issues/1603. Working on a fix.

**Comment by harp-intel** [comment]
Issue addressed in release 2.7.2.
